### PR TITLE
Added Pl3xmap support

### PR DESCRIPTION
**Pull request by ahdg6**
Port from https://github.com/Tartaros-Gamer/Pl3xMap-GriefDefender
Inspired by this repo:https://github.com/pl3xgaming/Pl3xMap-GriefPrevention

This PR added support for [Pl3xMap](https://github.com/pl3xgaming/Pl3xMap) and Solved https://github.com/bloodmc/GriefDefender/issues/376

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,81 @@
...
+* [CustomItems(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/36128/) ]
+* [EliteMobs]: [ [Spigot](https://www.spigotmc.org/resources/40090/) | [Github](https://github.com/MagmaGuy/EliteMobs) ]
+* [MMOItems(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/39267/) ]
+* [McMMO]: [ [Spigot](https://www.spigotmc.org/resources/64348/updates) | [Github](https://github.com/mcMMO-Dev/mcMMO) ]
+* [MyPet]: [ [Spigot](https://www.spigotmc.org/resources/mypet.12725/) | [Github](https://github.com/xXKeyleXx/MyPet) ]
+* [MythicMobs]: [ [Free](https://www.spigotmc.org/resources/5702/) | [Premium](https://www.spigotmc.org/resources/58415/) ]
+* [RevoltCrates(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/81681/) ]
+* [SimpleClans]: [ [Spigot](https://www.spigotmc.org/resources/71242/) | [Github](https://github.com/RoinujNosde/SimpleClans) ]
+
+#### Supported By Plugin Side
```
Comment (bloodmc) [inline README.md:41]: This section can go on a wiki page. It shouldn't be in the README as it is not related to GDHooks.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,81 @@
...
-* InsaneShops
-* MMOItems
-* McMMO
-* MyPet
-* MythicMobs
-* QuickShop
-* RevoltCrates
+GriefDefender currently support following plugins:  
+
+#### Supported By GDHook
```
Comment (bloodmc) [inline README.md]: Should be GDHooks

File: bukkit/src/main/java/com/griefdefender/hooks/config/category/Pl3xmapCategory.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+import com.griefdefender.api.GriefDefender;
+import com.griefdefender.api.claim.ClaimType;
+import com.griefdefender.api.claim.ClaimTypes;
+
+@ConfigSerializable
+public class Pl3xmapCategory {
+
+    @Setting("enabled")
+    @Comment("Set to true to enable GriefDefender Bluemap integration. (Default: true)")
```
Comment (bloodmc) [inline bukkit/src/main/java/com/griefdefender/hooks/config/category/Pl3xmapCategory.java]: Should be pl3xmap not bluemap in comment.

**Comment by bloodmc** [comment]
> What's more,May I join your discord to get the latest info about development?

Send an email to kanesoftware@outlook.com

File: README.md (hunk 1 above)
Comment (bloodmc) [reply README.md:41]: Actually, maybe it might be a good idea to have it all in one place. I'll think about it.

**Comment by ahdg6** [comment]
I have sent email to kanesoftware@outlook.com. Can you review this? https://github.com/bloodmc/GriefDefenderDefinitions/pull/1
Thanks.

**Comment by ahdg6** [comment]
Would you consider using https://github.com/riggraz/astuto (Self Hosted) or https://github.com/Plajer/feedbacky-project (Self Hosted or `Free Plan`) for managing roadmap and changelogs?
It will be easier to know what is happening to GD even someone doesn't join discord.

**Pull request merged**